src/homeworkNumber3/searchForASubstring.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <string.h>
+
+int countSubstringOccurrences(char S[], char S1[])
```

Comment:
```suggestion
int countSubstringOccurrences(char s[], char s1[])
```
Нарушение стиля именования переменных